addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/WebserviceHandler.java

```diff
@@ -66,7 +69,19 @@ public WebserviceHandler(Bridge bridge) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        // no code needed.
+        try {
+            if (channelUID.getId().contentEquals(CHANNEL_HOLIDAY)) {
+                logger.debug("HandleCommand recieved: {}", channelUID.getId());
+                if (command.toString() == "OFF") {
+                    isHoliday = false;
```

Comment:
Good Suggestion, however I'd like  change it to:
isHoliday=(command==OnOffType.ON)
Reason:
That way all possible command values are treated as desired with an Else and without the Need to catch any error (IMHO, see below).
command=ON results in True
command=OFF results in False
command anything else results in False!